src/deb.cc

```diff
@@ -27,3 +27,16 @@ Json::Value DebianManager::getInstalledPackages() {
   dpkg_program_done();
   return packages;
 }
+
+data::InstallOutcome DebianManager::install(const Uptane::Target &target) const {
+  std::string cmd = "dpkg -i ";
+  std::string output;
+  int status = Utils::shell(cmd + (path_ / "targets" / target.filename()).string(), &output);
+  if (status != 0) {
+    return data::InstallOutcome(data::INSTALL_FAILED, output);
+  }
+  Utils::writeFile(path_ / "targets/installed", target.sha256Hash());
```

Comment:
Would it be better to use INvStorage to store this information?

I suspect we'd need a new table in the SQL database, but I'm not sure how it relates to the storeInstalledVersions method we have today.

Reply:
Yes, I just noticed #552, so I will pass storage to packageManagerInterface